### ISSUE TITLE
[v2] unable to generate new app using v2 branch

### ISSUE THREAD

**Issue by toddw** [TrueCar/gluestick#675]
# Bug
Getting an error message when I generate a new app

## Steps to reproduce
1. `gluestick new my-app --dev ../gluestick/packages/gluestick`

## Current Result
```
[GlueStick] ERROR:  Cannot find module '/Users/twilliams/dev/package.json'
```

## Expected Result
No error, expect app to be generated

**Comment by toddw** [comment]
This was due to my creating the symbolic link to gluestick-cli incorrectly. Fixed that and it now works

**Comment by karna41317** [comment]
does not work still

**Comment by zamotany** [comment]
@karna41317 can you provide reproduction steps, which OS are you running, node and npm version??

**Comment by toddw** [comment]
@karna41317 Try using the published release candidate `gluestick-cli@1.0.0-rc.3`, then you wont have to use dev-mode

**Comment by ferrannp** [comment]
@karna41317 what branch are you using? If you point to `next`, the command should be:

```
gluestick new my-app --dev ../gluestick
```

But as @toddw said, you can just `npm install gluestick-cli@1.0.0-rc.3`
